### ISSUE TITLE
NATL60 Data Download

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/oceanbench#1]
**TLDR**
This module will download the data from an external server and store it to some local server. It will also archive it (e.g. filenames) so that it's easy to query and look through the data based on specific properties.

**Design Choice**

For development, we will download the data from an AWS server using DVC.

---
**Case Studies**: Some examples of other ways to download geoscience data.

* Aviso + Credentials
* MEOM ADAP Server
* Pangeo Recipes
* Copernicus

---
**Needed Functionality**

- [ ] #5
- [ ] Download SWOT Pseudo-Obs
- [ ] Download NADIR Pseudo-Obs
- [ ] Aviso Download + Credentials
- [ ] Pangeo Recipe
- [ ] Create verification
- [ ] Tutorial
